deploy/etryvoga/etryvoga.py

```
@@ -75,113 +75,116 @@ def format_time(time):
+                            case "ROCKET" | "ROCKET_FIRE":
+                                rockets_cached_data["states"][state_id] = region_data
+                            case "DRONE" | "RECON_DRONE":
+                                drones_cached_data["states"][state_id] = region_data
+                            case _:
+                                pass
+                        last_id = current_hex
+
+                    try:
+                        del explosions_cached_data["states"]["Невідомо"]
+                    except:
+                        pass
```

Ruff (0.8.2)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)